requirements/common.txt

```diff
@@ -10,13 +10,13 @@ django-import-export==0.5.1
 djangorestframework==3.9.3
 djangorestframework-expiring-authtoken==0.1.4
 django-cors-headers==1.3.1
-django-rest-auth[with_social]==0.9.2 
+django-rest-auth[with_social]==0.9.2
 django-ses==0.8.5
 docker-compose==1.21.0
 drfdocs==0.0.11
 drf-yasg==1.11.0
 kubernetes==10.0.1
-moto==1.3.8
+moto
```

Comment:
moto should be in dev

Reply:
nvm, since it was in common you can keep it here

Reply:
Okay!